packages/material-ui/src/AccordionSummary/AccordionSummary.js

```diff
@@ -140,7 +143,19 @@ AccordionSummary.propTypes = {
    * Override or extend the styles applied to the component.
    * See [CSS API](#css) below for more details.
    */
-  classes: PropTypes.object,
+  classes: chainPropTypes(PropTypes.object.isRequired, (props) => {
```

Comment:
```suggestion
  classes: chainPropTypes(PropTypes.object, (props) => {
```

Why required?

Reply:
Because it actually is (otherwise `withStyles` would not have worked). We just had it as optional because we used to generate the propTypes from it. But here it makes sense to make it obvious that we don't need to check if `props.classes` exists.

Reply:
Let's revert it to avoid confusion.